thumbor/filters/frame.py

```diff
@@ -88,7 +90,7 @@ def frame(self, callback, url):
         self.nine_patch_engine = self.context.modules.engine.__class__(self.context)
         self.storage = self.context.modules.storage
 
-        buffer = self.storage.get(self.url)
+        buffer = yield self.storage.get(self.url)
```

Comment:
@masom I think this one could be
```
buffer = yield gen.maybe_future(self.storage.get(self.url))
```
this way if the storage doesn't implement the gen. coroutine, it won't raise an error.